src/SFA.DAS.RequestApprenticeTraining.Api/Authentication/RoleNames.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Diagnostics.CodeAnalysis;
+
+namespace SFA.DAS.RequestApprenticeTraining.Api.Authentication
+{
+    [ExcludeFromCodeCoverage]
+    public static class RoleNames
+    {
+        public static string Default => "Default";
```

Comment:
@ChrisJWoodcock` PolicyNames.Default => nameof(Default)` & `RoleNames.Default =>"Default"` is better to stick to one convention for consistency instead of doing 2 different ways?